src/main/java/com/tfgm/TfgmApplication.java

```diff
@@ -5,20 +5,22 @@
 import java.net.URISyntaxException;
 import java.util.concurrent.TimeUnit;
 
+import com.tfgm.services.TramStopService;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 
-public final class App {
-  private App() {}
+@SpringBootApplication
+public class TfgmApplication {
 
   public static void main(String[] args)
       throws URISyntaxException, IOException, InterruptedException {
-
+    //    SpringApplication.run(TfgmApplication.class, args);
```

Comment:
This is better, but you should use this line as it will take care of running the program for you.

Reply:
It currently just breaks as it isn't connected to any db